### PR TITLE
[Documentation] Add details to default guidelines for all the libraries

### PR DESCRIPTION
**Pull request by rachelcarmena**
These guidelines appear in every README.

* How to build the library
* How to propose an improvement

### CODE REVIEW

File: docs/libraries/how-to-build-a-library.md [hunk 1]
```diff
@@ -4,6 +4,13 @@
 
 - JDK 8
 
+For [Arrow Fx](https://github.com/arrow-kt/arrow-fx):
+
+- Android SDK
+- `ANDROID_SDK_ROOT` environment variable for
```
Comment (aballano) [inline docs/libraries/how-to-build-a-library.md:10]: for? 😄

Comment (rachelcarmena) [reply docs/libraries/how-to-build-a-library.md]: Thanks for the review @aballano , fixed!!

File: docs/libraries/how-to-propose-an-improvement.md [hunk 2]
```diff
@@ -2,9 +2,42 @@
...
+* Include documentation via [Dokka](https://kotlinlang.org/docs/reference/kotlin-doc.html). Please, find examples in the existing code to follow the same pattern.
+* [Use Ank to validate for code snippets](https://github.com/arrow-kt/arrow-ank/blob/master/README.md)
+* Include tests that cover the proper cases
+
+When creating the pull request, ensure that these checks pass:
+
+* Required automatic checks:
+    * `arrow library: build`
+    * `arrow library: build documentation`
+* The approval by one of the members of the Arrow Community.
```
Comment (raulraja) [inline docs/libraries/how-to-propose-an-improvement.md:38]:
```suggestion
* The approval by 2 members of the Arrow Community.
```

File: docs/libraries/how-to-propose-an-improvement.md [hunk 3]
```diff
@@ -2,9 +2,42 @@
...
+* Include documentation via [Dokka](https://kotlinlang.org/docs/reference/kotlin-doc.html). Please, find examples in the existing code to follow the same pattern.
+* [Use Ank to validate for code snippets](https://github.com/arrow-kt/arrow-ank/blob/master/README.md)
+* Include tests that cover the proper cases
+
+When creating the pull request, ensure that these checks pass:
+
+* Required automatic checks:
+    * `arrow library: build`
+    * `arrow library: build documentation`
+* The approval by 2 members of the Arrow Community.
```
Comment (raulraja) [inline docs/libraries/how-to-propose-an-improvement.md:38]:
```suggestion
* The approval by 2 maintainers of the Arrow Community.
```

**Comment by rachelcarmena** [comment]
Thanks @raulraja , @aballano :raised_hands:

**Pull request merged**